plugins/dotnet-auth-core-identity/src/index.ts

```diff
@@ -106,10 +103,16 @@ class AuthCorePlugin implements dotnetTypes.AmplicationPlugin {
       destPath,
       filePath,
       context,
-      CsharpSupport.classReference({
-        name: `${resourceName}`,
-        namespace: `${resourceName}.Infrastructure`,
-      })
+      [
+        CsharpSupport.classReference({
+          name: `${resourceName}`,
```

Comment:
```suggestion
          name: `${resourceName}DbContext`,
```

Reply:
@overbit  Why??...

Reply:
as `${resourceName}` is not a class but a namespace, right?